lib/jsdom/living/nodes/HTMLFrameElement-impl.js

```diff
@@ -27,7 +27,7 @@ function loadFrame(frame) {
   // https://html.spec.whatwg.org/#process-the-iframe-attributes
   let url;
   const srcAttribute = getAttributeValue(frame, "src");
-  if (srcAttribute === null || srcAttribute === "") {
+  if (srcAttribute === "") {
```

Comment:
Why this change?

Reply:
`getAttributeValue` will return the empty string if the attribute doesn't exist, per [spec](https://dom.spec.whatwg.org/#concept-element-attributes-get-value).